### ISSUE TITLE
[How to Question]: LDAP Connection error

### ISSUE THREAD

**Issue by TestSubject1498** [babelfish-for-postgresql/babelfish_extensions#575]
### What happened?

TL;DR What are the officially supported means of authentication besides md5? I am struggling to find information on this in the babelfishpg.org documentation. md5 is not sufficient for my environment.

Version BABEL_2_1_STABLE (This version is not an option in the dropdown below)

Docker version 20.10.17, build 100c701 -- Ubuntu 20.04 as base image.

In pg_hba.conf that is produced when postgresql for babelfish is compiled, LDAP is listed as an available authentication method. 
From pg_hba.conf:

> \# METHOD can be "trust", "reject", "md5", "password", "scram-sha-256",
> \# "gss", "sspi", "ident", "peer", "pam", "ldap", "radius" or "cert".
> \# Note that "password" sends passwords in clear text; "md5" or
> \# "scram-sha-256" are preferred since they send encrypted passwords.

> \# TYPE  DATABASE        USER            ADDRESS                 METHOD
> \# "local" is for Unix domain socket connections only
> \# IPv4 local connections:
> host    all             all             0.0.0.0/0           ldap ldapserver=\<LDAP server FQDN\> ldapport=3268 ldaptls=0 ldapbasedn="DC=\<Domain\>" ldapbinddn="CN=<>,OU=Service Accounts,DC=\<Domain\>" ldapbindpasswd=\<pass\> ldapsearchattribute=\<account\>

As a sanity check I am able to connect to the babelfish db using pgadmin or psql. But if I use tsql, sqlcmd or ssms I receive the following error:

```
$ docker compose exec <container> bash -c 'sqlcmd -H localhost:1433 -U <user>'
Password: 
Sqlcmd: Error: Microsoft ODBC Driver 17 for SQL Server : pg_hba.conf entry specifies unsupported TDS authentication for host <ip address>, user <user>, database "bbf".
```

### Version

BABEL_1_X_DEV (Default)

### Extension

babelfishpg_tds

### Which flavor of Linux are you using when you see the bug?
Ubuntu 20.04

### Relevant log output

```shell
Sqlcmd: Error: Microsoft ODBC Driver 17 for SQL Server : pg_hba.conf entry specifies unsupported TDS authentication for host <ip address>, user <user>, database "bbf".
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

**Comment by kuntalghosh** [comment]
Babelfish TDS endpoint doesn't yet support any other authentication mechanism apart from
- md5
- password
- trust

While we're working on other authentication support, please use md5 authentication mechanism.

**Comment by TestSubject1498** [comment]
That's too bad. Thank you for the quick response. Are you able to tell me which auth mechanisms support is under development for babelfish?
